spec/models/donation_spec.rb

```diff
@@ -9,16 +9,27 @@
   it { is_expected.to validate_numericality_of(:amount).is_greater_than(0) }
 
   describe ".search" do
-    let(:smithie_black) { create(:donor, name: "Smithie Black") }
-    let(:bob_smith_white) { create(:donor, name: "Bob Smith-White") }
-    let(:john_smith) { create(:donor, name: "John Smith") }
-    let(:carl_grey) { create(:donor, name: "Carl Grey") }
+    before do
```

Comment:
I would setup this test like so:

```
before do
  create_list(:donor, 2)
end

let(:match_beginning) { create(:donation, create(:donor, "Smithie Black")) }
let(:match_middle) { create(:donation, create(:donor, "Bob Smith White")) }
let(:match_end) { create(:donation, create(:donor, "John Smith")) }

it { expect(described_class.search("smith").to contain_exactly(match_beginning, match_middle, match_end) }
```

That way it is clear which records are meant to match and not, and we can avoid the instance variable confusion. 😀